src/js/iov/IOV.js

```diff
@@ -78,13 +94,55 @@ export default class IOV {
       hostname = window.location.hostname;
     }
 
+    // if jwt extract required url parameters.
+    if (jwtUrl === true) {
+
+         //Url: clsp[s]-jwt://<sfs addr>[:9001]/<jwt>?Start=...&End=...
+        var qp_offset = url.indexOf(parser.pathname)+parser.pathname.length
+
+        var i = 0;
+        var qr_args = url.substr(qp_offset).split('?')[1];
+        var query = {
+        };
+
+        var pairs = qr_args.split('&');
+        for (var i = 0; i < pairs.length; i++) {
+            var pair = pairs[i].split('=');
+            query[decodeURIComponent(pair[0])] = decodeURIComponent(pair[1] || '');
+        }
```

Comment:
rather than parsing the query manually, what about using something like this:

https://github.com/sindresorhus/query-string